### PR TITLE
Todddjoh dev

### PR DESCRIPTION
**Pull request by tdjCisco**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -28,12 +28,47 @@ AC_PREFIX_DEFAULT(/usr/local/joy)
...
+AC_FUNC_REALLOC
+AC_FUNC_STRTOD
+AC_TYPE_MODE_T
+AC_TYPE_SSIZE_T
+AC_TYPE_INT8_T
+AC_TYPE_INT16_T
+AC_TYPE_INT64_T
+AM_PROG_CC_C_O
+AC_C_INLINE
+AC_CHECK_TYPES([ptrdiff_t])
```
Comment (tdjCisco) [inline configure.ac]: These could probably be consolidated but would rather deal with that in another PR

File: Makefile.am [hunk 2]
```diff
@@ -6,11 +6,6 @@ SUBDIRS=safe_c_stub lib bin
 joy_api:
 	(doxygen Doxyfile)
 
-cscope:
```
Comment (tdjCisco) [inline Makefile.am]: Centos7 had issues with this target. Removing for now.

Comment (bhudson33) [reply Makefile.am]: I use this target all the time. What was the specific issue?

Comment (hdbui) [reply Makefile.am]: I'm wondering if it's missing cscope from the base image.
I normally save off "find . -type f -follow -name "*.[ch]" -print > cscope.files" and run it as a script.

Comment (tdjCisco) [reply Makefile.am]: Actually cscope is a new default target for this autotools version. You can run make cscope as before.

File: configure.ac [hunk 3]
```diff
@@ -28,12 +28,47 @@ AC_PREFIX_DEFAULT(/usr/local/joy)
...
 AC_PROG_RANLIB
+AC_FUNC_STRNLEN
+AC_FUNC_MALLOC
+AC_FUNC_REALLOC
+AC_FUNC_STRTOD
+AC_TYPE_MODE_T
+AC_TYPE_SSIZE_T
+AC_TYPE_INT8_T
+AC_TYPE_INT16_T
+AC_TYPE_INT64_T
```
Comment (hdbui) [inline configure.ac]: Do we need AC_TYPE_INT32_T, AC_TYPE_UINT8_T ... AC_TYPE_UINT64_T?

Comment (tdjCisco) [reply configure.ac]: I will add them for completeness.

Comment (tdjCisco) [reply configure.ac]: Builds on MAC too

**Pull request merged**